### ISSUE TITLE
Provide recommendations to increase score

### ISSUE THREAD

**Issue by carmenbianca** [clearlydefined/clearlydefined#123]
Package maintainers may like to increase their score on ClearlyDefined. Currently, ClearlyDefined does not provide any recommendations to package maintainers on how they might achieve this. It would be great if this could be documented somewhere.

**Comment by nickvidal** [comment]
Good point @carmenbianca. Now that we have a new documentation website, we'll be working to expand it and include your suggestion.
